scripts/install/opt/seagate/cortx/motr/config/motr_cfg.sh

```diff
@@ -164,14 +164,24 @@ do_m0provision_action()
 
     msg "Configuring host [`hostname -f`]"
 
-    MD_DEVICE=$(lvs -o lv_path | grep "lv_raw_metadata" | head -1)
+    MD_DEVICE_NODE1=$(lvs -o lv_path | grep "lv_raw_metadata" | grep srvnode-1 | head -1)
```

Comment:
NODE1="srvnode-1"
    NODE-2="srvnode-2
MD_DEVICE_NODE1=$(lvs -o lv_path | grep "lv_raw_metadata" | grep $NODE1 | head -1)
    MD_DEVICE_NODE2=$(lvs -o lv_path | grep "lv_raw_metadata" | grep $NODE2 | head -1)

Let's change like this and ignore if srvnode-2 volumes are not present as it will be a valid case in LDR R2